lessons/beginners/comparisons/index.md

```diff
@@ -153,6 +153,11 @@ což je výraz, podle kterého se budeme rozhodovat.
 Za podmínkou je dvojtečka.
 Potom následují příkazy, které se provedou, pokud je podmínka pravdivá.
 Všechny jsou odsazeny o čtyři mezery.<br>
+
+> [note]
+> Pro odsazení textu doprava stačí použít klávesu <kbd>Tab</kbd>.
+> Pro návrat odsazení zpátky poté kombinaci kláves <kbd>Shift</kbd>+<kbd>Tab</kbd>.
```

Comment:
Před zmáčknutím <kdb>Tab</kbd> ale musíš vybrat nějaký text, ideálně přes více řádků. Jinak to (aspoň v některých editorech) jen vloží mezery na místo kurzoru

A v rámci jednotnějšího stylu prosím napiš `použij` nebo `zmáčkni` místo `stačí použít`.

Reply:
Díky, myslel jsem tím posunutí kurzoru, ne odsazení textu (tj. než začnou psát, odsadí si to zleva na správnou úroveň).

Tím pádem např. "Pro odsazení kurzoru" nebo "Pro zvýšení odsazení"?

Reply:
Aha, takhle!
Co takhle něco jako `Pro odsazení textu zmáčkni na začátku řádku klávesu <kbd>Tab</kbd>.`

Reply:
Možná už to řeším moc do detailu, ale takhle mi to trochu zní, jako bychom je nabádali nejprve napsat ten kód vždycky úplně vlevo a pak ho teprve odsadit, což je trochu zvláštní :D "Pro posunutí kurzoru o jednu úroveň odsazení doprava" je asi nejpřesnější, ale to už zní dost hekticky.

Každopádně když už to je odsazené a dá se enter, tak editory typicky to odsazení na dalším řádku udrží, takže to není takový problém. Hlavně, když pochopí, že mají použít tab a ne na každém řádku spamovat 4x mezerník, to byla ta hlavní zpráva, kterou jsem chtěl předat.

Reply:
```
> [note]
> Čtyři mezery neznamenají, že musíš čtyřikrát zmáčknout mezerník!
> K odsazení použij klávesu <kbd>Tab</kbd>, která vloží správný počet mezer.
> (Pokud ne, nemáš správně nastavený editor – podívej se do lekce o instalaci.)
> Pomocí <kbd>Shift</kbd>+<kbd>Tab</kbd> můžeš odsazení zase zmenšit.
>
> A ani <kbd>Tab</kbd> není vždycky potřeba.
> Pokud napíšeš řádek s `if` bez chyby, některé editory za tebe další řádek odsadí automaticky.
```